Bottom: 6c68d208273b143e17de8aff5ebfff6c80a36e25
Top:    4c0969ea3613d27f8003d68146ec6c3796156c21
Author: Ian Kent <raven@themaw.net>
Date:   2013-07-30 13:33:35 +0800

Refresh of autofs-5.0.7-add-dumpmap-to-file.patch

---

diff --git a/include/automount.h b/include/automount.h
index c2d39f9..15b1128 100644
--- a/include/automount.h
+++ b/include/automount.h
@@ -649,7 +649,7 @@ static inline FILE *open_fopen_wx(const char *path)
 
 #if defined(O_CLOEXEC) && defined(SOCK_CLOEXEC)
 	if (cloexec_works != -1) {
-		f = fopen(fd, "wxe");
+		f = fopen(path, "wxe");
 		if (f != NULL) {
 			check_cloexec(fileno(f));
 			return f;
